plugins/obs-ffmpeg/cmake/dependencies.cmake

```diff
@@ -2,7 +2,7 @@
 if(OS_WINDOWS OR OS_MACOS)
   set(ffmpeg_version 6)
 else()
-  set(ffmpeg_version 5.1)
+  set(ffmpeg_version 4.4)
```

Comment:
Was this a bad rebase? PR mentions the version should be 5.1?

Reply:
That was because there is no Ubuntu 22 for my ARM64 VM anymore and Ubuntu 23 supports 5.1, but for GitHub Actions we need to drop down to 4.4 for Ubuntu 22.